### PR TITLE
#161290918 Enable Users Like Comments and Replies

### PR DESCRIPTION
**Pull request by Frediflexta**
#### What does this PR do?
* Enables users like comments and also like replies they find appealing 
#### Description of Task to be completed?
⁃ Write tests
⁃ Create a table relationship between likes, comments, reply and users model
⁃ Enable users like a comment
⁃ Enable users like a reply
⁃ Refactor get all comment of an article to include the number of likes
⁃ Refactor get all replies of a comment include the number of likes
#### How should this be manually tested?
* Clone repo
* Run `npm install` to get all the packages
* Sign up users following #11 
* Create an article following instructions on #13 
* Create comments and replies following the instructions on #35 
* Like comments to an article by sending a POST to `http://localhost:3000/api/v1/comments/:commentId/likes`
* Like a reply to a comment by sending a POST to `http://localhost:3000/api/v1/replies/:replyId/likes`
*N/B: If a user likes a comment/reply and hits the route again, they unlike that comment/reply
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#161290918
#### Screenshots (if appropriate)
screen shot 2018-11-21 at 9 03 54 pm
screen shot 2018-11-21 at 9 04 00 pm
screen shot 2018-11-21 at 9 04 46 pm
screen shot 2018-11-21 at 9 04 50 pm

#### Questions:
N/A

### CODE REVIEW

**Review by oluseyianani — APPROVED** [review]
Looks good @Frediflexta

**Review by ovieokeh — APPROVED** [review]
Awesome work. Thanks for renaming workers to queries.

**Review by desaawa — COMMENTED** [review]
Please resolve conflicts

**Review by ajimae — APPROVED** [review]
Nice one @Frediflexta, especially on the reply functionality. You did well.

**Comment by desaawa** [comment]
Hi @Frediflexta, tests are failing.

File: logs/info.log [hunk 1]
```diff
@@ -0,0 +1,55 @@
+[Thu Nov 22 2018 17:54:23 GMT+0100 (WAT)] : undefined
```
Comment (desaawa) [inline logs/info.log:1]: We shouldn't be having these files here on Github, should we?

**Review by desaawa — APPROVED** [review]

File: logs/info.log (hunk 1 above)
Comment (Frediflexta) [reply logs/info.log:1]: I just checked with a teammate, apparently, it shouldn't, we will remove it, actually, It's a feature that a teammate worked on so I was unaware of about it. So sorry about that.

**Pull request merged**